client/src/views/Models/components/MetadataPanel.vue

```diff
@@ -1,32 +1,53 @@
 <template>
-  <div v-if="metadata" class="metadata-pane-container">
-    <div v-for="(item, key) in metadata" :key="key">
-      {{item}}
-    </div>
-    <!-- <div class="metadata-item">
-      <div class="key">Name</div>
-      <div>{{metadata.name}}</div>
-    </div>
-    <div class="metadata-item">
-       <div class="key">Description</div>
-      <div>{{metadata.description}}</div>
-    </div>
-    <div class="metadata-item">
-       <div class="key">Created</div>
-      <div>{{metadata.created}}</div>
-    </div>
-    <div class="metadata-item">
-       <div class="key">Source</div>
-      <div>{{metadata.source}}</div>
-    </div>
-    <div class="metadata-item">
-       <div class="key">Version</div>
-      <div>{{metadata.version}}</div>
-    </div>
-    <div class="metadata-item">
-       <div class="key">Knowledge</div>
-      <div><a class="text-link" :href="metadata.knowledge">{{metadata.knowledge}}</a> </div>
-    </div> -->
+  <div class="metadata-panel">
+    <details v-for="(datum, index) in metadata" :key="index">
+
+      <template v-if="isTypeModel(datum)">
+        <summary :title="datum.uid">Model</summary>
+        <h6>Variables</h6>
+        <p>{{ datum.variables.join(', ') }}</p>
+        <h6>Parameters</h6>
+        <p>{{ datum.parameters.join(', ') }}</p>
+        <h6>Initial Conditions</h6>
+        <p>{{ datum.initial_conditions.join(', ') }}</p>
+      </template>
+
+      <template v-else-if="isTypeCode(datum)">
+        <summary :title="datum.uid">Code</summary>
+        <h6>Reference</h6>
+        <p>{{ datum.global_reference_id.type }} {{ datum.global_reference_id.id }}</p>
+        <h6>Files</h6>
+        <ol>
+          <li
+            v-for="(file, index) in datum.file_ids"
+            :key="index"
+            :title="file.uid"
+          >{{ file.name }} ({{ file.path }})</li>
+        </ol>
+      </template>
+
+      <template v-else-if="isTypeDocuments(datum)">
+        <summary :title="datum.uid">Documents</summary>
+        <details v-for="(doc, index) in datum.documents" :key="index">
+          <summary :title="doc.uid">{{ doc.bibjson.title }}</summary>
+          <h6>Source</h6>
+          <p><a :href="doc.bibjson.website.url">{{ doc.bibjson.type }}</a></p>
```

Comment:
I think we should embed this url in the title and we can get rid of this source field. Something like:

`        <summary :title="doc.uid"><a :href="doc.bibjson.website.url">{{ doc.bibjson.title }}</a></summary>
`